### ISSUE TITLE
I tried to run the sample code in your repository but cannot see the contact at incoming call

### ISSUE THREAD

**Issue by tungdt-gem** [Yuedevelopment/calldemoios10#1]
Hi @Yuedevelopment , Thank you so much for sharing your example code at here. 

While trying to run your sample, I follow the same steps as described in your blog, but I don't know why the extension not working. I cannot debug the method beginRequest... and cannot see how this work. Could you please help?

Thanks and best regard

**Comment by Yuedevelopment** [comment]
Can your download my demo to OK?

**Comment by Yuedevelopment** [comment]
@tungdt-gem this is  how iOS 10 works

**Comment by tungdt-gem** [comment]
yes, I downloaded and run complete but alway show message 有错误. i run in ios 10

**Comment by Yuedevelopment** [comment]
@tungdt-gem You do not have permission

**Comment by tungdt-gem** [comment]
how to set permission?
